### PR TITLE
COMMERCE-11957 - When including a dot as part of a Specification Key, the Specification Facet does not work

### PR DESCRIPTION
**Pull request by jaydawu**
https://liferay.atlassian.net/browse/COMMERCE-11957
Resent from https://github.com/liferay-commerce/liferay-portal/pull/4010

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceFacets.testcase [hunk 1]
```diff
@@ -246,6 +246,108 @@ definition {
 			locator1 = "CommerceAccelerators#PRODUCT_CARD_NAME");
 	}
 
+	@description = "This is a test for COMMERCE-11957. As a buyer, I want to filter products using Specification Facet widget by specification value and the results are visible on the Search Results widget"
+	@priority = 5
+	test AssertProductsAreFilteredBySpecificationValue {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceFacets.testcase:251]: As the focus of the test is to not be able to have a dot in the specification key, please find a name more suitable for the covered scenario

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceFacets.testcase [hunk 2]
```diff
@@ -534,6 +534,56 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-11957. Admin cannot view the specification entry's key is set with dots"
+	@priority = 5
+	test CannotViewSpecificationKeyIsSetWithDots {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceFacets.testcase:539]:
```suggestion
	test CannotAddSpecificationKeyWithDots {
```

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
Hi [@victoroscuro](https://github.com/victoroscuro), there are no relevant errors to this pull, please force-forward it, thanks!

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**